test/e2e/nsx_security_policy_test.go

```diff
@@ -259,572 +307,15 @@ func TestSecurityPolicyNamedPortWithoutPod(t *testing.T) {
 
 	psb, err := testData.deploymentWaitForNames(defaultTimeout, nsWeb, labelWeb)
 	t.Logf("Pods are %v", psb)
-	assertNil(t, err, "Error when waiting for IP for Pod %s", webA)
+	assert.NoError(t, err, "Error when waiting for IP for Pod %s", webA)
 	err = testData.waitForCRReadyOrDeleted(defaultTimeout, SP, nsWeb, securityPolicyCRName, Ready)
-	assertNil(t, err, "Error when waiting for Security Policy %s", securityPolicyCRName)
+	assert.NoError(t, err, "Error when waiting for Security Policy %s", securityPolicyCRName)
 
 	// Check NSX resource existing
 	err = testData.waitForResourceExistOrNot(nsWeb, common.ResourceTypeSecurityPolicy, securityPolicyCRName, true)
-	assertNil(t, err)
+	assert.NoError(t, err)
 	err = testData.waitForResourceExistOrNot(nsWeb, common.ResourceTypeRule, ruleName0, true)
-	assertNil(t, err)
+	assert.NoError(t, err)
 	err = testData.waitForResourceExistOrNot(nsWeb, common.ResourceTypeRule, ruleName1, true)
-	assertNil(t, err)
-}
-
-/*
-// TestSecurityPolicyNamedPort0 verifies that the traffic of security policy when named port applied.
```

Comment:
please don't remove these temporary removed code, we might add it back since we are now running e2e test in WCP env.

Reply:
We can use `testing.Skip` function to skip the test case.